Populist Legal Power/common/laws/yMoG_PLP_citizenship.txt

```diff
@@ -81,7 +89,7 @@ law_ethnostate = {
 		
 		country_disallow_government_work_violent_hostility_bool = yes
 		country_disallow_government_work_cultural_erasure_bool = yes
-		country_disallow_government_work_open_prejudice_bool = no
+		country_disallow_government_work_open_prejudice_bool = yes # no
```

Comment:
Note that this would mean that even Shared Heritage + Cultural Traits + State Religion would prevent government work unless they were in their homelands.
This is equivalent to also setting second rate citzens bool to yes as well, as with Ethnostate there are no second rate citizens.